src/main/java/io/github/cottonmc/energy/api/EnergyPacket.java

```diff
@@ -1,31 +0,0 @@
-package io.github.cottonmc.energy.api;
-
-/**
- * Kind of like an "EnergyStack". Represents energy in a movable or consumable form, ready to do work.
- */
-public class EnergyPacket {
-
-    public static final EnergyPacket EMPTY_PACKET = new EnergyPacket(DefaultEnergyTypes.ENERGY_TYPE_EMPTY, 0);
```

Comment:
I do think that even if we get rid of the general "packet" idea, having an EnergyStack-like for giving a type and an amount is still helpful. It could let people put NBT on energy and such, which seems like it'd be fun to make a mod around.

Reply:
I think that's well and truly out of scope for a mod like this, and something I'd hate to support when making industrial blocks.

Reply:
Also, I fully expect some wire transports to packetize their energy during transfer. Nobody's stopping people from doing that.

Reply:
Both fair points.